### PR TITLE
Refactors to hero cards and application list

### PR DESCRIPTION
**Pull request by t8**

### CODE REVIEW

**Review by 7i7o — APPROVED** [review]
tACK 👍

File: src/utils/format.ts [hunk 1]
```diff
@@ -7,8 +7,8 @@ export function formatStateVal(val: number) {
   if (val < 1000) {
     return val;
   } else if (val >= 1000 && val < 1_000_000) {
-    return (val / 1000).toFixed(1) + "K";
+    return (val / 1000).toFixed() + "K";
   } else if (val >= 1_000_000 && val < 1_000_000_000) {
```
Comment (7i7o) [inline src/utils/format.ts:11]: We should remove the `<1_000_000_000` so anything greater than a million gets shortened to M

**Comment by 7i7o** [comment]
@nicholaswma do you want to merge after your review?

**Comment by nicholaswma** [comment]
LGTM

**Pull request merged**